### DISCUSSION TITLE
csv_to_gdf error

### DISCUSSION THREAD

**Discussion by jllucero** [opengeos/leafmap#57] [Q&A] [answered]
Hello.
I'm trying to add facility locations to a layer.  My original file is excel.  I converted it to csv and tried to read it in using "csv_to_gdf," "csv_to_shp," and "csv_to_geojson." Using all three I get an error.  So I tried using "Map.add_xy_data(....)" but that gives me the following error. 
AttributeError: 'Map' object has no attribute 'add_xy_data'

What's strange is that I used this same process on another csv file using "csv_to_gdf" and it worked just fine.

I'd appreciate any suggestions.

Thank you.

**Comment by giswqs** [comment]
Please provide a small subset of your data for debugging

**Reply by jllucero** [reply]
[Data_v1.zip](https://github.com/giswqs/leafmap/files/6712355/Data_v1.zip)
Thank you very much.  Attached are a few data points.

**Comment by giswqs** [comment, marked as answer]
@jllucero The data you sent worked fine on my computer. 

![](https://i.imgur.com/Li5IEbS.gif)

**Reply by jllucero** [reply]
This is strange.  Using the same file that I sent  to you with this code, 
gdf = leafmap.csv_to_gdf(Data_v1, latitude="Latitude", Longitude="Longitude")

I get the following error,
TypeError: csv_to_gdf() got an unexpected keyword argument 'Longitude'

However, I am able to load the file just like you did by using the toolbox on the map.

**Reply by giswqs** [reply]
The parameters are lower case.

**Reply by jllucero** [reply]
This is really embarrassing.  I don't know how long I've stared at the code.  You are correct.  It works fine now.  Thank you!
